### PR TITLE
Automate Profile screen

### PR DESCRIPTION
**Pull request by bilalawan321**

### CODE REVIEW

**Comment by bilalawan321** [comment]
@jawad-khan can you please review this PR.?

File: tests/android/pages/android_profile.py [hunk 1]
```diff
@@ -0,0 +1,209 @@
+"""
```
Comment (jawad-khan) [inline tests/android/pages/android_profile.py:1]: No need or this.

File: tests/android/pages/android_profile.py [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+            self.driver,
+            android_elements.profile_screen_user_language
+        )
+
+        if language_variable:
+            return self.global_contents.wait_and_get_element(
+                self.driver,
+                android_elements.profile_screen_user_language
+            )
+        else:
```
Comment (jawad-khan) [inline tests/android/pages/android_profile.py:116]: No need for this.

File: tests/android/pages/android_profile.py [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+            self.driver,
+            android_elements.profile_screen_limited_view
+        )
+
+        if limited_view:
+            return self.global_contents.wait_and_get_element(
+                self.driver,
+                android_elements.profile_screen_limited_view
+            )
+        else:
```
Comment (jawad-khan) [inline tests/android/pages/android_profile.py:142]: No need for this.

File: tests/android/tests/test_android_login.py [hunk 4]
```diff
@@ -119,5 +119,5 @@ def test_upgrade_app(self, set_capabilities, setup_logging):
         Verifies that user can upgrade app
         """
 
-        global_contents = Globals(setup_logging)
-        assert global_contents.upgrade_target_app(set_capabilities)
+        # global_contents = Globals(setup_logging)
```
Comment (jawad-khan) [inline tests/android/tests/test_android_login.py:122]: if not needed, then remove it.
If commented intentionally then write comments for it.

File: tests/android/tests/test_android_profile.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+                "Profile" as Title
+                Edit
+                Profile Image
+                User Name
+            Verify that Profile screen will show following contents for limited profile:
+                Age limit text
+                Account settings Button
+            Verify that Profile screen will show following contents for limited profile:
+                location
+                Laguage (if selected)
```
Comment (jawad-khan) [inline tests/android/tests/test_android_profile.py:49]: Language maybe.

**Review by jawad-khan — APPROVED** [review]
Just small changes otherwise looks good.

File: tests/android/tests/test_android_login.py (hunk 4 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_login.py:122]: mistakenly pushed, I am uncommenting this.

File: tests/android/tests/test_android_profile.py (hunk 5 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_profile.py]: fixed

File: tests/android/pages/android_profile.py (hunk 3 above)
Comment (bilalawan321) [reply tests/android/pages/android_profile.py]: fixed

File: tests/android/pages/android_profile.py (hunk 2 above)
Comment (bilalawan321) [reply tests/android/pages/android_profile.py]: fixed

**Pull request merged**